docs/system/wires_fuses.md

```diff
@@ -68,6 +113,19 @@ A Fuse is an important component in electrical systems, used to prevent excessiv
 
 ### Branch
 
-In case of branches where multiple lines are drawn from the main wire as shown in the Fig.3, fuse has to be placed on each individual branch lines. Failure in doing so might result in excessive current in individual lines of the branch beyond their limit even when total current in the main line is within the limit. i.e, Fuse $F$ might have higher current rating than fuse $F_1$. In this case, even though when fuse $F$ doesn't trip and current is within the safe limit of the main line, fuse $F_1$ might trip due to excessive current in branch 1 beyond the limit of branch 1 line. Absence of $F_1$ in this case leads to damage of the systems.
+In case of branches where multiple lines are drawn from the main wire as shown in the Fig.3, a fuse has to be placed on each individual branch lines. Failure in doing so might result in excessive current in individual lines of the branch beyond their limit even when total current in the main line is within the limit. i.e, Fuse $F$ might have higher current rating than fuse $F_1$. In this case, even though when fuse $F$ doesn't trip and current is within the safe limit of the main line, fuse $F_1$ might trip due to excessive current in branch 1 beyond the limit of branch 1 line. Absence of $F_1$ in this case leads to damage of the systems.
 
 <fig-caption src="system/fuse_branches.svg" caption="Position of fuse in branches" num="3" />
+
+
+## Grounding
+
+Ground defines a common voltage level as a reference point, to which all other voltages are measured. This can be done by connecting all negative **or** positive sides to the same line, which can be a rod driven into the earth or the chassis of a car. To prevent accidents by faulty wires or electrical machines, it is common to connect non-electrical but conductive parts (e.g. machine housing) to ground as well. In case of a fault and a short circuit, current flows through the housing and into the earth rather than the body and through the legs into earth.
```

Comment:
```suggestion
The ground defines a common voltage level as a reference point, to which all other voltages are measured. This can be done by connecting all negative **or** positive sides to the same line, which can be a rod driven into the earth or the chassis of a car. To prevent accidents by faulty wires or electrical machines, it is common to connect non-electrical but conductive parts (e.g. machine housing) to ground as well. In case of a fault and a short circuit, current flows through the housing and into the earth rather than the body and through the legs into earth.
```